docs/topics/migrations.txt

```diff
@@ -114,8 +114,8 @@ Django projects without the need for a full database.
 Workflow
 ========
 
-Working with migrations is simple. Make changes to your models - say, add
-a field and remove a model - and then run :djadmin:`makemigrations`::
+Django will create migrations for you. Make changes to your models - say, add a
```

Comment:
I'd say "Django can create migrations for you." - it won't if you don't tell it, and you have the option to create them yourself too.